win32ss/gdi/ntgdi/freetype.c

```diff
@@ -4990,7 +4914,7 @@ NtGdiGetFontFamilyInfo(HDC Dc,
     /* Enumerate font families in the global list */
     IntLockGlobalFonts;
     Count = 0;
-    if (! GetFontFamilyInfoForList(&LogFont, Info, &Count, Size, &FontListHead) )
+    if (! GetFontFamilyInfoForList(&LogFont, Info, &Count, Size, NULL, &FontListHead) )
```

Comment:
Remove the extra space after '!' and before "GetFont..."

Reply:
Done.